src/lightning_lite/utilities/distributed.py

```diff
@@ -136,10 +136,10 @@ def sync_ddp(result: Tensor, group: Optional[Any] = None, reduce_op: Optional[Un
 
     # WA for HPU. HPU doesn't support Long types, forcefully set it to float
     if module_available("habana_frameworks.torch.utils.library_loader"):
-        from habana_frameworks.torch.utils.library_loader import is_habana_available
+        from habana_frameworks.torch.utils.library_loader import is_habana_avaialble
 
         if (
-            is_habana_available()
+            is_habana_avaialble()
```

Comment:
#15000 did not uncover this because HPU tests are not set to run for Lite changes. This is somewhat expected because Lite does not support HPUs, but there's this HPU-specific code here that should be moved.

Reply:
Is that really a typo that you changed the code to?

Reply:
Yes, they actually have this typo in the main library. This was already noticed on other HPU PRs :)